pkg/util/deployment/deployment.go

```diff
@@ -40,6 +40,12 @@ import (
 const (
 	// The revision annotation of a deployment's replica sets which records its rollout sequence
 	RevisionAnnotation = "deployment.kubernetes.io/revision"
+	// DesiredReplicasAnnotation is the desired replicas for a deployment recorded as an annotation
+	// in its replica sets.
+	DesiredReplicasAnnotation = "deployment.kubernetes.io/desired-replicas"
```

Comment:
Yes for now it is, but if we develop another type of controller that manages scaling across multiple replica sets and wants to have similar behavior I wonder if we would want it to use the same annotation or a different one.  As I understand it, the annotation's purpose is to record what fraction of a whole this replica set is intended to represent.